server.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask
+from flask_socketio import SocketIO, emit
+
+# constants
+PORT = 8080
+HEIGHT = 690
+WIDTH = 1280
+
+# create Flask object
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'secret!'
+socketio = SocketIO(app, cors_allowed_origins='*')
+
+# server class
+class Server:
+    def __init__(self, height, width):
+        self.board = []
+        # initialize empty board
+        for i in range(0,height):
+            row = []
+            for j in range(0,width):
+                row.append(0)
+            self.board.append(row)
+# instantiate server class for board state
+server = Server(HEIGHT,WIDTH)
+
+@socketio.on('connect')
+def on_connect():
+    print("connected to websocket")
+    emit('broadcast-board', server.board)

```

Comment:
We probably don't want to emit the board to everyone when a new client joins right?

Why don't we just expose a different REST API (maybe '/canvas' GET) as an endpoint to get the current board state?
Note that this would NOT be a websocket endpoint.  It may be wasteful, but for the sake of having both REST and websocket, let's just do that.

Reply:
Removed emit() from the function. I will add the endpoint in a different PR